pkg/apis/cnsoperator/config/cnsregistervolume_crd.yaml

```diff
@@ -10,62 +15,79 @@ spec:
+          metadata:
+            type: object
+          spec:
+            description: CnsRegisterVolumeSpec defines the desired state of CnsRegisterVolume
+            properties:
+              accessMode:
+                description: AccessMode contains the actual access mode the volume
+                  backing the CnsRegisterVolume has. AccessMode must be specified
+                  if VolumeID is specified.
+                type: string
+              diskURLPath:
+                description: 'DiskUrlPath is URL path to an existing block volume
+                  to be imported into Project Pacific cluster. VolumeID and DiskUrlPath
+                  cannot be specified together. DiskUrlPath is explicitly used for
+                  block volumes. AccessMode need not be specified and will be defaulted
+                  to "ReadWriteOnce". This field must be in the following format:
+                  Format: https://<vc_ip>/folder/<vm_vmdk_path>?dcPath=<datacenterName>&dsName=<datastoreName>
+                  Ex: https://10.192.255.221/folder/34a9c05d-5f03-e254-e692-02004479cb91/        vm2_1.vmdk?dcPath=Datacenter-1&dsName=vsanDatastore
+                  This is for a 34a9c05d-5f03-e254-e692-02004479cb91/vm2_1.vmdk file
+                  under datacenter "Datacenter-1" and datastore "vsanDatastore".'
+                type: string
+              pvcName:
+                description: Name of the PVC
+                type: string
+              volumeID:
+                description: VolumeID indicates an existing vsphere volume to be imported
+                  into Project Pacific cluster. If the AccessMode is "ReadWriteMany"
+                  or "ReadOnlyMany", then this VolumeID can be either an existing
+                  FileShare (or) CNS file volume backed FileShare. If the AccessMode
+                  is "ReadWriteOnce", then this VolumeID can be either an existing
+                  FCD (or) a CNS backed FCD. VolumeID and DiskUrlPath cannot be specified
+                  together.
+                type: string
+            required:
+            - pvcName
+            type: object
+          status:
+            description: CnsRegisterVolumeStatus defines the observed state of CnsRegisterVolume
+            properties:
+              error:
+                description: The last error encountered during import operation, if
+                  any. This field must only be set by the entity completing the import
+                  operation, i.e. the CNS Operator.
+                type: string
+              registered:
+                description: Indicates the volume is successfully registered. This
+                  field must only be set by the entity completing the register operation,
+                  i.e. the CNS Operator.
+                type: boolean
+            required:
+            - registered
+            type: object
+        type: object
     served: true
     storage: true
+    subresources:
```

Comment:
With status added as subResource, it is preventing operations like modify/update/add on status field. Please remove the subresources section